screens/StudySpaceDetailScreen/LiveSeatingMapList.js

```diff
@@ -14,39 +13,38 @@ const styles = StyleSheet.create({
 
 class LiveSeatingMapList extends Component {
   static propTypes = {
-    token: PropTypes.string,
-    survey: PropTypes.shape(),
+    maps: PropTypes.arrayOf(PropTypes.shape()),
   };
   static defaultProps = {
-    token: "",
-    survey: null,
+    maps: [],
   };
 
-  static mapStateToProps = state => ({
+  static mapStateToProps = (state: Object) => ({
     token: state.user.token,
   });
   static mapDispatchToProps = () => ({});
+
+  renderMapInfo = ({ id, name, capacity, occupied }) => (
+    <BodyText key={id}>
+      {name}: {capacity - occupied} seats free (total: {capacity})
+    </BodyText>
+  );
+
   render() {
-    const { survey } = this.props;
-    const hasMaps =
-      survey &&
-      survey.maps &&
-      Array.isArray(survey.maps) &&
-      survey.maps.length > 0;
+    const { maps } = this.props;
+    const hasMaps = maps && Array.isArray(maps) && maps.length > 1;
+    // no need breakdown if only 1 map
```

Comment:
```suggestion
    // No breakdown needed if there is only one map in the survey
```
Just being grammatically pedantic in case we need to use this comment in the future